src/tachimawari/joint/node/joint_manager.cpp

```diff
@@ -83,14 +97,20 @@ bool JointManager::torque_enable(bool enable)
 
 bool JointManager::torque_enable(const std::vector<Joint> & joints, bool enable)
 {
-  if (joints.size()) {
-    for (const auto & joint : joints) {
-      return control_manager->write_packet(
-        joint.get_id(), protocol_1::MX28Address::TORQUE_ENABLE, enable);
-    }
+  if (std::any_of(
+      joints.begin(), joints.end(), [&](Joint joint) {
+        return control_manager->write_packet(
```

Comment:
Is this expression proper?
Because I think this one should return negation of the `write_packet` method calls.

Reply:
my bad, updated